cpp/perspective/CMakeLists.txt

```diff
@@ -509,7 +509,7 @@ set(PYTHON_SOURCE_FILES ${SOURCE_FILES}
 )
 
 set(WASM_SOURCE_FILES ${SOURCE_FILES}
```

Comment:
should remove whole block if not used anymore

Reply:
[used here](https://github.com/finos/perspective/pull/2339/files#diff-50e5b576dee1bef31c114f1bef14d219054509aa2da37937228763998409692dR556)